### DISCUSSION TITLE
Merge own photos with photos shared by partner in user timeline

### DISCUSSION THREAD

**Discussion by denispol** [immich-app/immich#4281] [Q&A]
Hi,

The partner sharing feature works great, I am wondering however if there is the possibility to be able to see both the user's photos as well as the partner's in the main timeline? This would allow unified search, map, faces, etc.

If this is not possible, could this be a potential feature? I think that this is really useful for families and couples that want to share all their photos and videos.

**Comment by ghost** [comment]
sharing face data is the ONE thing no other photo manager does, and its the ONE thing I would like to see added :)

anything thing might be in the `share partner` items to share.. photo, faces, could allow certain items

**Comment by ubrfkr** [comment]
I would also find immich enormously more useful if my partner and I could see all our photos together in one place. The current workflow to see our pictures from a certain date is to look at my own library, then click over to my partner's library and look at the same date. I'm surprised this isn't a more highly requested feature.

**Reply by ghost** [reply]
I don't really want joint view. Sharing of faces yes 😀 

I think if you really want that much of a linked account have you thought of just creating one user account and you both log into that one account?? Everything would then all be in one folder with the same albums, maps etc ??

**Reply by fivestones** [reply]
@oziee But some people want to be able to see the photos either way. I.e., usually I would want a view with my photos and my wife's photos merged. But sometimes, I (or she) would want to be able to see just our own (or just the other's) photos on a timeline.

**Comment by fivestones** [comment]
It would be great to just have a filter with the timeline, that let you optionally see a merged view or just yours or just your partner's photos.

If you've used Calibre or paperless-ngx or any number of other apps that show items in a timeline, this would be similar to how they let you show (in the case of Calibre) all authors' books in the same book list or pick out one or more to show alone, or (in the case of paperless-ngx) documents from all correspondents in a single timeline, or only documents from a particular correspondent.

**Comment by OlegLoewen** [comment]
For me, this is essential and would love to see this feature. This is exactly how it is done in Google Photos and my partner and I like it a lot.

**Reply by andymarden** [reply]
Upvote - sometimes I wonder how people look at features and link them to how people are actually going to use software.

It is self evident to me that there is an overall use case of:
- An existing photo folder structure that is accessed in multiple ways - some legacy, some not (eg DLNA server)
- The folder structure belongs to a family or org collectively
- The folder structure being segregated for legacy and usage in other ways to have logical albums by sub-folder
- A group of users would want to be able to have full immich functionality on this collective structure - maps, timeline, etc
- Each user also to be able to have their own external structure as well as internal to share

This would lead to certain conclusions - eg included in maps when shared; not having to have an external library added to one users external path (why cannot it just not be set as an external library by the admin and we're done?)

It seems the external libraries functionality is viewed as a separate bolt on but really they're not. I would encourage the devs to look at things through this lens.

**Comment by bo0tzz** [comment]
See #2846

**Comment by denispol** [comment]
#4974 is the first PR to address it, thank you!
